pkgs/by-name/sh/shoko-webui/package.nix

```diff
@@ -0,0 +1,53 @@
+{
+  stdenvNoCC,
+  fetchFromGitHub,
+  nodejs,
+  pnpm,
```

Comment:
Most likely you need to change `pnpm` to `pnpm_9`. Since #371832, it points to `pnpm_10` instead of `pnpm_9`.

Reply:
Seems to work with 10 after a rebase

Reply:
Indeed it works, thanks for trying it out, you can mark this as resolved.